spec/shr_allergy_map.txt

```diff
@@ -1,30 +1,30 @@
 Grammar:	Map 5.1
-Namespace:	cimi.allergy
+Namespace:	shr.allergy
 Target:		FHIR_STU_3
 
 
-AdverseSensitivityToSubstanceStatement maps to http://hl7.org/fhir/us/core/StructureDefinition/us-core-allergyintolerance:
+AllergyIntolerance maps to http://hl7.org/fhir/us/core/StructureDefinition/us-core-allergyintolerance:
 //	PersonOfRecord maps to patient
 //	SourceOfInformation.Value maps to asserter 
 //	Signed.Value.EntityOrRole maps to recorder 
 	_Entry.CreationTime maps to assertedDate
-	AdverseSensitivityTopic.AllergenIrritant maps to code
-	AdverseSensitivityTopic.Type maps to type
-	AdverseSensitivityTopic.SubstanceCategory maps to category
-	VerificationStatus maps to verificationStatus
+	FindingResult maps to code
+	AllergyIntoleranceType maps to type
+	SubstanceCategory maps to category
+	FindingStatus maps to verificationStatus
 	ClinicalStatus maps to clinicalStatus
 	Criticality maps to criticality
 	Onset maps to onset[x]
 	MostRecentOccurrenceTime maps to lastOccurrence
-	AdverseReaction.Manifestation.Severity maps to reaction.severity
-	AdverseReaction.CausalAttribution.RouteIntoBody maps to reaction.exposureRoute
-	AdverseReaction.CausalAttribution.Substance maps to reaction.substance
-	AdverseReaction.Manifestation.Value maps to reaction.manifestation
-	AdverseReaction.CommentOrDescription maps to reaction.description
-	AdverseReaction.Onset maps to reaction.onset
-
-
-NoAdverseSensitivityToSubstanceStatement maps to http://hl7.org/fhir/us/core/StructureDefinition/us-core-allergyintolerance:
+	AllergyIntoleranceReaction.SubstanceCode maps to reaction.substance
+	AllergyIntoleranceReaction.Manifestation maps to reaction.manifestation
+	AllergyIntoleranceReaction.BeginDateTime maps to reaction.onset
+	AllergyIntoleranceReaction.Severity maps to reaction.severity
+	AllergyIntoleranceReaction.RouteIntoBody maps to reaction.exposureRoute
+	AllergyIntoleranceReaction.CommentOrDescription maps to reaction.description
+
+/*
+NoAdverseSensitivityToSubstance maps to http://hl7.org/fhir/us/core/StructureDefinition/us-core-allergyintolerance:
```

Comment:
I can't find this class in shr.allergy. Should this throw an error?

Reply:
Should be okay.  It's commented out.  The full shr_allergy_map.txt shows the start of the comment block on lne 26 and ends on at line 53 (EOF).